qoffer-domain/src/main/groovy/life/qbic/portal/portlet/customers/affiliation/create/CreateAffiliationOutput.groovy

```diff
@@ -16,4 +16,10 @@ interface CreateAffiliationOutput extends UseCaseFailure {
      * @since 1.0.0
      */
     void affiliationCreated(Affiliation affiliation)
+
+    /**
+     * This method informs the output that the afiliation was created successfully
+     * @param message
+     */
+    void successNotification(String message)
```

Comment:
Why is this in this PR?

Reply:
I added this because my IDE told me the method is called although it is not part of the interface. I considered it as a minor fix that is not necessary to be included in an own PR. The code only works because the method is actually provided although not required by the interface definition.

Reply:
hm i still think this should not be part of this PR though. And it must be part of the interface or removed, otherwise I don't see why it is still there.

Reply:
I can see your point. But I think we really need to fix it to prevent future problems. I stumbled over it multiple times now and afaik we use the method to display a success to the user.